README.md

```diff
@@ -79,6 +79,41 @@ User network home directories are provided by [SoftNAS from AWS Marketplace](htt
 
 ### Defining new environment
 
+#### Auth0
+
+1. Create a new tenant:
+
+    1. Log-in to Auth0
+    2. Click on your user
+    3. In the drop-down menu click "Create tenant"
+         * Tenant domain: include the environment name, if not the platform
+
+2. Create an application:
+
+    1. In the side-bar click "Applications"
+    2. Click "Create Application"
+         * Name: AWS
```

Comment:
Does this name needs to be `AWS Console`? You mention this in step 3.2 later on right?